logging/cluster-logging-deploying.adoc

```diff
@@ -1,21 +1,21 @@
 :context: cluster-logging-deploying
 [id="cluster-logging-deploying"]
-= Installing cluster logging
+= Installing OpenShift Logging
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 
-You can install cluster logging by deploying
-the Elasticsearch and Cluster Logging Operators. The Elasticsearch Operator
-creates and manages the Elasticsearch cluster used by cluster logging. 
-The Cluster Logging Operator creates and manages the components of the logging stack. 
+You can install OpenShift Logging by deploying
+the Elasticsearch and cluster logging operators. The Elasticsearch Operator
```

Comment:
"Cluster Logging"

Reply:
@ewolinetz Sorry - I don't understand which string I should change to "Cluster Logging." Would you clarify?